src/matchers/toBeAccessible/index.spec.ts

```diff
@@ -109,4 +109,21 @@ test.describe.parallel('toBeAccessible', () => {
     expect(duration).toBeGreaterThan(1000)
     expect(duration).toBeLessThan(1500)
   })
+
+  test(`default report filename`, async ({ page }) => {
+    test.fail()
```

Comment:
Now that you are catching the error for `toBeAccessible`, this test shouldn't be failing.

```suggestion
```